src/components/Book.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+
+const Book = props => {
+  const { id, title, category } = props;
+  return (
+    <tr>
+      <td>{id}</td>
+      <td>{title}</td>
+      <td>{category}</td>
+    </tr>
+  );
+};
+
```

Comment:
The `Book` component should accept **Book object** as a prop, not id, title, and category ([screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/Book/AccceptBookObjectAsProp.png))